backend/rates/history.go

```diff
@@ -20,15 +20,15 @@ var (
 	// Values are copied from https://api.coingecko.com/api/v3/coins/list.
 	// TODO: Replace keys with coin.Code.
 	geckoCoin = map[string]string{
-		"BTC": "bitcoin",
-		"LTC": "litecoin",
-		"ETH": "ethereum",
+		"btc": "bitcoin",
+		"ltc": "litecoin",
+		"eth": "ethereum",
 		// Useful for testing with testnets.
-		"TBTC": "bitcoin",
-		"RBTC": "bitcoin",
-		"TLTC": "litecoin",
-		"TETH": "ethereum",
-		"RETH": "ethereum",
+		"tbtc": "bitcoin",
+		"rbtc": "bitcoin",
+		"tltc": "litecoin",
+		"teth": "ethereum",
+		"reth": "ethereum",
```

Comment:
a little bit below:

```
"eth-erc20-dai0x6b17": "dai",
"eth-erc20-sai0x89d2": "sai",
```

instead of the dai that is there.

Reply:
:see_no_evil: done
thanks!

Reply:
(ordered alphabetically)